lib/react_client/react_interop.dart

```diff
@@ -113,7 +113,7 @@ class ReactElementStore {
 @JS()
 @anonymous
 class ReactElement {
-  external ReactElementStore get _store;
+  external ReactElementStore get _store; // ignore: unused_element
```

Comment:
I don't really know what it is @jacehensley-wf ... let's follow up on that at a later date.